### PR TITLE
Adding azure monitor metrics addon image to agent baker

### PR DESCRIPTION
**Pull request by rashmichandrashekar**
**What type of PR is this?**
Adding a new repo link for azure monitor metrics addon

### CODE REVIEW

**Comment by rashmichandrashekar** [comment]
@robbiezhang - I am unable to assign this you for review - could you pls take a look?

**Comment by rashmichandrashekar** [comment]
@vishiy @gracewehner @bragi92 @Sohamdg081992 - FYI

**Review by vishiy — APPROVED** [review]

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -137,6 +137,20 @@
         "ciprod08102022"
       ]
     },
+    {
```
Comment (robbiezhang) [inline vhdbuilder/packer/components.json:140]: why they are not multi-arch?

Comment (rashmichandrashekar) [reply vhdbuilder/packer/components.json:140]: @robbiezhang - We are not planning to support multi-arch for public preview.

Comment (rashmichandrashekar) [reply vhdbuilder/packer/components.json:140]: @robbiezhang - Updated this for kube-state-metrics. Thanks for pointing it out.

**Review by robbiezhang — APPROVED** [review]

**Pull request closed without merging**